### ISSUE TITLE
Misleading "Example-Xxxx" section titles

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode-docs#1443]
Whenever drilling into the "Extension Authoring" chapter I am always confused by the usage of the word "Example":

![2018-02-20 10-35-53](https://user-images.githubusercontent.com/1898161/36416634-dfdc1b68-1629-11e8-94bf-1c7c4f17edc9.png)

In my (non-native) understanding of the English language "Example-Debuggers" is something completely different than "Example: Debuggers".

The former means "a list of example debuggers" the latter alludes to "another example: how to write a debugger extension".

This applies to the other titles as well but the most disturbing is the "Example-Debuggers" title.

Could we improve these titles, e.g. by using "Example: Debugger Extensions"?

/cc @isidorn

**Comment by isidorn** [comment]
Agree.
Anothering thing is that the same prefix of each section makes it hard to destinguish between the seciont. I would prefer if it were called
Debuggers Example
Language Server Example
Or insteand of the Example I would use the word Extension.

**Comment by gregvanl** [comment]
I agree the TOC naming is a little awkward but we came to the current "Example - *" pattern as a workaround for not being able to have a third level TOC. Ideally, all four examples would go under an Examples sub node in the Extensions TOC. 
We tried using "Example: *" but this broke TOC generation.
@weinand How about we change "Example - Debuggers" to "Example - Debug Adapter"? This is then similar to "Example - Language Server".
